fda-model/paireddata/UncertainPairedData.cs

```diff
@@ -9,8 +9,17 @@ namespace paireddata
 {
     public class UncertainPairedData: IPairedDataProducer, ICategory, ICanBeNull
     {
+        #region Fields 
         private double[] _xvals;
         private IDistribution[] _yvals;
+        #endregion
+
+        #region Properties 
+        public string XLabel { get; }
```

Comment:
this metadata seems to apply to both paired data and uncertain paired data, if we made it an object would it be beneficial for both?

Reply:
thinking narrowly, it looks like we plan to only accept uncertain paired data (and a distribution for flow freq). From that perspective, I am not sure how useful it would be to attach metadata to paired data. 

for general use, I see that we could pull this information out into an object and use the object for both. In that case, would we want overloaded constructors for the paired data so that we do not need to pass metadata to a paired data object upon sampling an uncertain paired data object with each iteration?

Reply:
you are right. we arent storing paired data (currently). i forgot about that.  An object might be easier in the long run. what you have now is great and we can stage a refactor if necessary some other day.

Reply:
and the paired data is created via the sample of uncertain data, we would just have to update that logic to pass in the metadata, were we to do this action in the future.

Reply:
Deal. Thanks, Will.